dotnet/src/IntegrationTests/TestSettings/TavilyConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace SemanticKernel.IntegrationTests.TestSettings;
+
+[SuppressMessage("Performance", "CA1812:Internal class that is apparently never instantiated",
```

Comment:
Suggest using pragma instead.

```suggestion
[SuppressMessage("Performance", "CA1812:Internal class that is apparently never instantiated",
```